app/instance-initializers/ember-a11y.js

```diff
@@ -1,10 +1,12 @@
 // This initializer exists only to make sure that the following
 // imports happen before the app boots.
 import Ember from 'ember';
-import { registerKeywords } from 'ember-a11y/ember-internals';
 import pivotRouteIdentifier from 'ember-a11y/utils/pivot-route-identifier';
 
+//{{htmlbars}}
+import { registerKeywords } from 'ember-a11y/ember-internals';
```

Comment:
ahh, I didn't realize this was the only thing there.  Maybe we could just have separate initializers? I'm not _opposed_ to the `broccoli-string-replace` stuff to remove this, but I'd prefer to not have to do it...